lighthouse-extension/app/src/lighthouse-background.js

```diff
@@ -124,19 +147,30 @@ window.runLighthouseForConnection = function(connection, url, options, requested
   lighthouseIsRunning = true;
   updateBadgeUI(url);
 
-  // Run Lighthouse.
-  return Runner.run(connection, runOptions)
-    .then(result => {
-      lighthouseIsRunning = false;
-      updateBadgeUI();
-      filterOutArtifacts(result);
-      return result;
-    })
-    .catch(err => {
-      lighthouseIsRunning = false;
-      updateBadgeUI();
-      throw err;
-    });
+  return enableOtherChromeExtensions(false)
+      .then(_ => Runner.run(connection, runOptions)) // Run Lighthouse.
+      .then(result => {
+        lighthouseIsRunning = false;
+        updateBadgeUI();
+        filterOutArtifacts(result);
+
+        // Deliver results even if enabling extensions fails.
+        enableOtherChromeExtensions(true).catch(err => {
+          log.warn('Chrome', `Could not disable some extensions. ${err.message}`);
```

Comment:
`disable` -> `enable`? This seems like a big problem if it ever happens...worth bubbling up to the "Report Error" level?